src/app/features/home/components/advertisement/sell-card/sell-card.component.ts

```diff
@@ -13,7 +13,6 @@ export class SellCardComponent implements OnInit {
 
   constructor(private readonly cardService: CardService) { }
 
-
   ngOnInit(): void {
     this.cards = this.cardService.getCards('sellCards.json')
```

Comment:
Components shouldn't know anything about resource URL. Create getSellCards method and call it